chat-agent/tools/leave/leave_tools.py

```
@@ -0,0 +1,503 @@
+    return headers
+
+
+def _leave_error_from_response(response: httpx.Response) -> dict:
+    if response.status_code in (200, 201, 204):
+        return {}
+    try:
+        body = response.json()
+        msg = body.get("message") if isinstance(body, dict) else None
+        if msg:
+            return {"error": msg}
+    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)